config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+
+  root 'blog#index'
+
+  get 'blogs' => 'blog#index'
+  get 'blogs/:id' => 'blog#show'
+  get 'blogs/new' => 'blog#new'
```

Comment:
you'll want the alias's for these routes as well. 

, as: 'blogs'
, as 'blog'
, as: 'new_blog'